src/libraries/System.Net.Http.Json/src/System/Net/Http/Json/HttpClientJsonExtensions.Delete.cs

```diff
@@ -67,6 +107,16 @@ public static partial class HttpClientJsonExtensions
             return DeleteFromJsonAsyncCore<TValue>(taskResponse, options, cancellationToken);
         }
 
+        /// <summary>
+        /// Sends a DELETE request to the specified Uri and returns the value that results from deserializing the response body as JSON in an asynchronous operation.
+        /// </summary>
+        /// <param name="client">The client used to send the request.</param>
+        /// <param name="requestUri">The Uri the request is sent to.</param>
+        /// <param name="type">The type of the object to deserialize to and return.</param>
+        /// <param name="context">Source generated JsonSerializerContext used to control the deserialization behavior.</param>
```

Comment:
Can JsonSerializerContext be described in natural language? Maybe something like:

```suggestion
        /// <param name="context">Source generated JSON serializer context used to control the deserialization behavior.</param>
```

I didn't suggest a `see cref` because the parameter type already shows up next to the parameter name in MS Docs.